users/views.py

```diff
@@ -133,9 +133,10 @@ def user_login(request):
             user.backend = 'django.contrib.auth.backends.ModelBackend'
             ip = get_ip(request)
             logger.info('user %s @ %s logged in' % (str(user), ip))
-            one_hour = 60 * 60
-            request.session.set_expiry(one_hour)
-            logger.info('user %s set session timeout one hour' % str(user))
+            hours = int(get_config('session_expiry', 'expiry'))
```

Comment:
If you are not familiar with singleton pattern, please read this and discuss with me or @nickelcheng 
http://stackoverflow.com/questions/6760685/creating-a-singleton-in-python